scopes/component/merging/merge-cmd.ts

```diff
@@ -22,18 +22,18 @@ export class MergeCmd implements Command {
   extendedDescription = `merge changes of the remote head into local. when on a lane, merge the remote head of the lane into the local.
 if no ids are specified, all pending-merge components will be merged. (run "bit status" to list them).
 optionally use '--abort' to revert the last merge. to revert a lane merge, use "bit lane merge-abort" command.
-${WILDCARD_HELP('merge')}`;
+${WILDCARD_HELP('merge')}`; // @david does this command auto-snap by default? If so we must document it here
```

Comment:
Yes. This command auto-snap by default. You can opt-out by `--no-snap`.

Reply:
cheers, added that to the description